website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step08-apply-helm-chart.sh

```diff
@@ -1,6 +1,9 @@
 # Logout of helm registry to perform an unauthenticated pull against the public ECR
 helm registry logout public.ecr.aws
 
+# Set context to the newly created cluster.
+aws eks update-kubeconfig --name ${CLUSTER_NAME} --region ${AWS_DEFAULT_REGION}
```

Comment:
I was under the impression that `eksctl` sets this context for you after finishing cluster creation. Can you double-check that this is not the case and that we explicitly need to do this?

Reply:
I was thinking the same but it didn't set the cluster to current context. I created a bash script to setup karpenter cluster and tried 5 separate demo with different configs but hit the same issue because it only used context from the first cluster.

```
Error: Kubernetes cluster unreachable: Get "https://<cluster-id>.gr7.us-west-2.eks.amazonaws.com/version": dial tcp: lookup <cluster-id>.gr7.us-west-2.eks.amazonaws.com: no such host
```
Adding update-kubeconfig helped solve the above error.
```
aws eks update-kubeconfig --name ${CLUSTER_NAME} --region ${AWS_DEFAULT_REGION}
```

Reply:
@shamsher31 I've tested this a few times and I see this coming through my `eksctl` run

```console
2023-07-31 17:47:11 [ℹ]  serviceaccount "kube-system/aws-node" already exists
2023-07-31 17:47:11 [ℹ]  updated serviceaccount "kube-system/aws-node"
2023-07-31 17:47:11 [ℹ]  daemonset "kube-system/aws-node" restarted
2023-07-31 17:47:12 [ℹ]  building managed nodegroup stack "eksctl-joinnis-test5-nodegroup-joinnis-test5-ng"
2023-07-31 17:47:12 [ℹ]  deploying stack "eksctl-joinnis-test5-nodegroup-joinnis-test5-ng"
2023-07-31 17:47:12 [ℹ]  waiting for CloudFormation stack "eksctl-joinnis-test5-nodegroup-joinnis-test5-ng"
2023-07-31 17:47:42 [ℹ]  waiting for CloudFormation stack "eksctl-joinnis-test5-nodegroup-joinnis-test5-ng"
2023-07-31 17:48:39 [ℹ]  waiting for CloudFormation stack "eksctl-joinnis-test5-nodegroup-joinnis-test5-ng"
2023-07-31 17:50:29 [ℹ]  waiting for CloudFormation stack "eksctl-joinnis-test5-nodegroup-joinnis-test5-ng"
2023-07-31 17:50:30 [ℹ]  waiting for the control plane to become ready
2023-07-31 17:50:30 [✔]  saved kubeconfig as "/Users/joinnis/.kube/config"
```

You can see the line that says that it is saving the updated context into my `KUBECONFIG` so it looks like it always merges it on my side. Do you see any errors that you can show from doing an `eksctl cluster create` that would indicate that it isn't merging the context?